Makefile

```diff
@@ -12,5 +26,11 @@ updaterdk:
 	go get go.viam.com/rdk@latest
 	go mod tidy
 
+FFmpeg:
+	git clone https://github.com/FFmpeg/FFmpeg.git --depth 1 --branch release/6.1
+
+build-ffmpeg: FFmpeg
+	cd FFmpeg && ./configure $(FFMPEG_OPTS) && make -j$(shell nproc) && make install
```

Comment:
Use of `nproc` here will not work on MacOS.

Reply:
Can we please detect if we are on osx & run the equivalent command for osx?

Reply:
nvm, `nproc` fill is actually working on macos.